edb/lib/std/30-jsonfuncs.edgeql

```diff
@@ -153,6 +153,12 @@ CREATE CAST FROM anytuple TO std::json {
 };
 
 
+CREATE CAST FROM std::json TO anytuple {
+    SET volatility := 'STABLE';
+    USING SQL EXPRESSION;
+};
```

Comment:
@elprans Why were the tests passing without this?

Reply:
Because these are special casts (`FROM SQL EXPRESSION`), which are handled by the compiler directly and not via the generic dispatch.

Reply:
I don't think this answers my question.

Reply:
Is the answer that these formally defined casts are there just for the introspection?

Reply:
Yes, for introspection and completeness sake.